internal/api/api.go

```diff
@@ -64,6 +64,12 @@ func Run(ctx context.Context, logger zerolog.Logger, store db.Store) {
 	// More info: https://github.com/google/cel-spec
 	app.Post("/build-cel", webhookController.BuildCEL)
 
+	// Register Vehicle Subscription routes
+	vehicleSubscriptionController := controllers.NewVehicleSubscriptionController(store, logger)
+
+	app.Post("/subscriptions", vehicleSubscriptionController.AssignVehicleToWebhook)
```

Comment:
Have the tokenId be in the path eg. subscriptions/:tokenID 
Reason: consistency with other API's, can't remember the exact RESTful rule but in my mind it just looks more RESTful if you have the object you're modifying in the path.

Reply:
you could even do something like
`/subscritptions/:vehicleTokenID/event/:eventID`